docker/build-rust-all.sh

```diff
@@ -4,7 +4,7 @@
 set -e
 
 # Build all the rust release binaries
-cargo build --release \
+RUSTFLAGS="--cfg tokio_unstable" cargo build --release \
```

Comment:
If we don't provide this flag to `cargo build`, do these crates fail to build? If so, this is a problem. It'll completely break building and running locally.

Reply:
This is a good question. AFAIK, this flag just compiles additional APIs into the binary and does not actually modify existing ones. 
IMHO having these tools available now is important enough to allow this. 

I am planning on checking for performance regression with regard to this change.

If needed this can be hidden by adding some flags to the compile script, allowing for debug and release builds.

Reply:
Local builds are not broken because of:

--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -19,3 +19,6 @@ xclippy = [
   "-Aclippy::enum-variant-names",
   "-Aclippy::self-named-constructors",
 ]
+
+[build]
+rustflags = ["--cfg", "tokio_unstable"]